### DISCUSSION TITLE
Not able to build the session (ConnectionError: Unable to connect after 10 attempts)

### DISCUSSION THREAD

**Discussion by MrNemes1s** [kaskada-ai/kaskada#619] [Q&A]
I am trying to setup Kaskanda in my local machine ( WSL2 Debian running on Windows 11) and I am not able to initiate the session 
Before Initiating I have pip installed the Kaskanda and imported LocalBuiled . This is the first time I am using kaskanda .. Did I Miss any step ? 

![image](https://github.com/kaskada-ai/kaskada/assets/53710275/20475aed-13ae-42f9-a6a8-57a9502390ab)

**Comment by epinzur** [comment]
Hi. Thanks for trying Kaskada.  We haven't had many users with your setup.  

Could you please look in the log files listed and send us the contents, so that we can better troubleshoot the issue?

**Reply by MrNemes1s** [reply]
![image](https://github.com/kaskada-ai/kaskada/assets/53710275/f9859fc5-bbf0-400a-97c2-8522206b485c)

**Reply by epinzur** [reply]
thanks for this.  what version of Debian are you running in WSL2?

**Reply by epinzur** [reply]
I believe that `bullseye` or newer will be required to run Kaskada

**Reply by MrNemes1s** [reply]
Latest version and also I tried installing with AWS amazon linux 2 and faced same issue
![image](https://github.com/kaskada-ai/kaskada/assets/53710275/c56219b5-1d37-423c-9c2a-d7cedf0e2818)
